@here/harp-mapview/lib/MapView.ts

```diff
@@ -1797,19 +1834,51 @@ export class MapView extends THREE.EventDispatcher {
             this.m_forceCameraAspect !== undefined ? this.m_forceCameraAspect : width / height;
         this.setFovOnCamera(this.m_options.fovCalculation!, height);
 
-        const kMinNear = 0.1;
-        const kMultiplier = 50.0;
-        const kFarOffset = 200.0;
-
-        let nearPlane = Math.max(kMinNear, this.m_camera.position.z * 0.1);
-        let farPlane = nearPlane * kMultiplier + kFarOffset;
-
-        if (this.m_options.farPlaneEvaluator !== undefined) {
-            this.camera.getWorldDirection(this.m_tempVector3);
-            const angle = THREE.Math.radToDeg(this.m_tempVector3.angleTo(EYE_INVERSE));
-            const nearFarPlane = this.m_options.farPlaneEvaluator(this, angle, nearPlane, farPlane);
-            nearPlane = Math.max(kMinNear, nearFarPlane.near);
-            farPlane = Math.max(nearPlane + kFarOffset, nearFarPlane.far);
+        let nearPlane: number = 0;
+        let farPlane: number = 0;
+
+        if (this.projection.type === ProjectionType.Spherical) {
```

Comment:
Someone commened it some time ago.

constructor is already huuge, maybe move it to `setupNearFarPlanes ?

Reply:
this code is not in the constructor, it's in updateCamera.